project/DataflowStreamingExampleProjectBuild.scala

```diff
@@ -0,0 +1,46 @@
+/* 
```

Comment:
Again, I think this ^^ should be called GoogleCloudDataflowExampleProjectBuild.scala